common/OpTestHMC.py

```diff
@@ -360,31 +357,34 @@ def recover_lpar(self, src_mg_system, dest_mg_system, stop_lpm=False, timeout=30
         log.info("LPAR failed to recover at managed system %s" % src_mg_system)
         return False
 
-    def get_adapter_id(self, mg_system, loc_code):
+    def get_adapter_id(self, mg_system, loc_code, remote_hmc=None):
+        hmc = remote_hmc if remote_hmc else self
```

Comment:
assign variable in single line not looks good to me please have like below 
```
if remote_hmc:
   hmc = remote_hmc
else:
   hmc = self
  ```

Reply:
sure no hard feeling, just looking on code maintenance front as going with block it looks more clear :)